packages/kbn-esql-ast/src/parser/__tests__/function.test.ts

```diff
@@ -7,8 +7,8 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import { getAstAndSyntaxErrors as parse } from '../ast_parser';
-import { Walker } from '../walker';
+import { getAstAndSyntaxErrors as parse } from '..';
```

Comment:
We could always just rename `getAstAndSyntaxErrors` to `parse`.

Reply:
Yes, we should do that. I just didn't want to do too many changes in other people codebases.